src/test/datascience/notebook/executionService.vscode.test.ts

```diff
@@ -576,7 +576,10 @@ suite('DataScience - VSCode Notebook - (Execution) (slow)', () => {
                         `,
             { index: 0 }
         );
-        const cells = vscodeNotebook.activeNotebookEditor?.document.cells!;
+        if (!vscodeNotebook.activeNotebookEditor) {
```

Comment:
I can see `cells` are undefined, possibly meaning there are no notebooks.
This could happen when we open the start page (at which point the notebook is no longer active).
Again fix in other PR https://github.com/microsoft/vscode-jupyter/pull/3976